src/common/GeometryCommon/GeometryDecorator.tsx

```diff
@@ -139,9 +139,8 @@ export class GeometryDecorator implements Decorator {
 
   // Iterate through the geometry and point lists, extracting each geometry and point, along with their styles
   // Adding them to the graphic builder which then creates new graphics
-  // TODO: Add the ability to support text rendering
   public createGraphics(context: DecorateContext): RenderGraphic | undefined {
-    const builder = context.createGraphicBuilder(GraphicType.Scene);
+    const builder = context.createGraphicBuilder(GraphicType.Scene, undefined, "GeometryGraphic");
```

Comment:
This needs to be an Id64String. See documentation for `createGraphicBuilder`.

Reply:
Would it be correct to use TransientIdSequence.next here?

Reply:
Yes. If `createGraphics` produces new graphics on every call from `decorate`, prefer to obtain and reuse one transient Id; if it caches its graphics for reuse, just obtain a transient Id here.

Reply:
Do edges draw correctly now? Can you add a comment to explain to users why you associated an Id with the graphics?

Reply:
Yes the edges are drawing correctly without the offset after those changes. And yes I can add a comment too.

Reply:
Does the comment I added provide enough information to the user? I don't fully understand why the Id resolves that edge issue, so I'm not sure how much more detail would be necessary in that comment

Reply:
Sorry. I'd say something like "Specifying an Id for the graphics tells the display system that all of the geometry belongs to the same entity, so that it knows to make sure the edges draw on top of the surfaces."